leaf/utils/pagination.py

```diff
@@ -47,6 +57,16 @@ async def previous(
     async def page(
         self, interaction: discord.Interaction, button: discord.ui.Button
     ) -> None:
+        if self.author and interaction.user != self.author:
```

Comment:
Should we add some error handling here for cases where the interaction may not be valid anymore? Maybe surround with a try/except and log an exception or something.

Reply:
@bsrecker In what case would it be invalid?

Reply:
That I don't know. Theoretically it should never happen, but best practice is probably to proactively prevent the bot from crashing if it did happen.